tasks/winbuildscripts/junitupload.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = "Stop"
+$Password = ConvertTo-SecureString "dummyPW_:-gch6Rejae9" -AsPlainText -Force
+New-LocalUser -Name "ddagentuser" -Description "Test user for the secrets feature on windows." -Password $Password
+
+$Env:Python3_ROOT_DIR=$Env:TEST_EMBEDDED_PY3
+
+if ($Env:TARGET_ARCH -eq "x64") {
+    & ridk enable
+}
+& $Env:Python3_ROOT_DIR\python.exe -m  pip install -r c:\mnt\requirements.txt
+
+$UT_BUILD_ROOT=(Get-Location).Path
+$Env:PATH="$UT_BUILD_ROOT\dev\lib;$Env:GOPATH\bin;$Env:Python3_ROOT_DIR;$Env:Python3_ROOT_DIR\Scripts;$Env:PATH"
+
+Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/download/datadog-ci_win-x64.exe" -OutFile "datadog-ci.exe"
```

Comment:
Don't use `Invoke-WebRequest` to download files in scripts
https://github.com/PowerShell/PowerShell/issues/2138

```suggestion
$url = 'https://github.com/DataDog/datadog-ci/releases/latest/download/datadog-ci_win-x64.exe'
(New-Object System.Net.WebClient).DownloadFile($url, "datadog-ci.exe")
```